src/newbank/server/model/Transaction.java

```diff
@@ -0,0 +1,69 @@
+package newbank.server.model;
+
+import java.util.Calendar;
+import java.util.Date;
+
+public class Transaction {
```

Comment:
The class looks well designed, and now I have to think about how this could be integrated with the MOVE command that I have implemented.

Reply:
PAY/Transfer is also account to account. Move is a special case of that.   
Pay <same_person> <account_type> <Amount> 

Currently I am keeping this. But I guess I will have to make a validation later that the "PAY" command doesnt do move. This can be something for the backlog. Let me know if this should be restricted now itself. 

Transaction from account to account is already handled in the "SHOWTRANSACTIONS"  
Login as John, 
PAY John Savings 10 
SHOWTRANSACTIONS 

Would like to see how this will work with MOVE.